.github/workflows/deploy.yml

```
@@ -0,0 +1,21 @@
+name: Deploy Release
+on:
+  push:
+    branches:
+      - "automatic-release-tag"
```

Branch does not exist anymore, please change to develop (as staging system)
```suggestion
      - "develop"
```